heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -232,6 +237,20 @@ function process_message()
             if type(info.subsessionCounter) == "number" then
                 msg.Fields.subsessionCounter.value = info.subsessionCounter
             end
+
+            if type(info.subsessionStartDate) == "string" then
+                local t = dt.rfc3339:match(info.subsessionStartDate)
+                if t then
+                    msg.Fields.subsessionTimestamp.value = dt.time_to_ns(t)
+                else -- some dates are not RFC compliant
+                    t = broken_date:match(info.subsessionStartDate)
+                    if t then
+                        msg.Fields.subsessionTimestamp.value = dt.time_to_ns(t)
+                    else
+                        msg.Fields.subsessionTimestamp.value = cts
```

Comment:
I don't think we should fall back to creationTimestamp here - we already have that value in a separate field, so we should just leave out the subsessionTimestamp if it's unavailable (or unparseable).